dnf-docker-test/features/steps/updateinfo_steps.py

```diff
@@ -0,0 +1,99 @@
+         Tag        Default value
+    ============== ====================
+    Title          Default title of Id
+    Type           security
+    Description    Default description of Id
+    Summary        Default summary of Id
+    Severity       Low
+    Solution       Default solution of Id
+    Rights         Chuck Norris
+    Issued         2017-01-01 00:00:01
+    Updated        2017-01-01 00:00:01
+    Reference      none
+    Package        none
+    ============== ===============
+
+    Examples:
+
+    .. code-block:: gherkin
+
+       Feature: Defining updateinfo in a repository
+
+         @setup
+         Scenario: Repository base with updateinfo defined
+              Given repository "base" with packages
+                 | Package | Tag     | Value |
+                 | foo     | Version | 2     |
+                 | bar     | Version | 2     |
+              And updateinfo defined in repository "base"
+                 | Id            | Tag         | Value                     |
+                 | RHSA-2017-001 | Title       | foo bar security update   |
+                 |               | Type        | security                  |
+                 |               | Description | Fixes buffer overflow     |
+                 |               | Summary     | Critical bug is fixed     |
+                 |               | Severity    | Critical                  |
+                 |               | Solution    | Update to the new version |
+                 |               | Rights      | Copyright 2017 Baz Inc    |
+                 |               | Reference   | CVE-2017-0001             |
+                 |               | Reference   | BZ123456                  |
+                 |               | Package     | foo-2                     |
+                 |               | Package     | bar-2                     |
+
+    .. note::
+
+       Specifying Version or Release in Package tag is not necessary, however
+       when multiple RPMs matches the string the last one from the sorted
+       list is used.
+    """
+    HEADINGS_GROUP = ['Id', 'Tag', 'Value']
+    UPDATEINFO_TAGS_REPEATING = ['Package', 'Reference']
+    UPDATEINFO_TAGS = ['Title', 'Type', 'Description', 'Solution', 'Summary', 'Severity', 'Rights', 'Issued', 'Updated'] + \
+                      UPDATEINFO_TAGS_REPEATING
+    updateinfo_table = table_utils.parse_skv_table(ctx, HEADINGS_GROUP, UPDATEINFO_TAGS, UPDATEINFO_TAGS_REPEATING)
+
+    # verify that modifyrepo_c is present
+    modifyrepo = which("modifyrepo_c")
+    ctx.assertion.assertIsNotNone(modifyrepo, "modifyrepo_c is required")
+
+    # prepare updateinfo.xml content
+    repodir = updateinfo_utils.get_repo_dir(repository)
+    updateinfo_xml = updateinfo_utils.get_updateinfo_xml(repository, updateinfo_table)
+
+    # save it to the updateinfo.xml file and recreate the repodata
+    tmpdir = tempfile.mkdtemp()
+    fw = open('%s/updateinfo.xml' % tmpdir, 'w')
```

Comment:
* use context manager for this (`with open() as fw:`)
* Use `os.path.join(tmpdir, "updateinfo.xml")`